server/rest/tiles.py

```diff
@@ -186,6 +195,36 @@ def _getTilesInfo(self, item, imageArgs):
         except TileGeneralException as e:
             raise RestException(e.message, code=400)
 
+    def _setContentDisposition(self, item, contentDisposition, mime, subname, params):
+        """
+        If requested, set the content disposition and a suggested file name.
+
+        :param item: an item that includes a name.
+        :param contentDisposition: either 'inline' or 'attachemnt', otherwise
+            no header is added.
+        :param mime: the mimetype of the output image.  Used for the filename
+            suffix.
+        :param subname: a subname to append to the item name.
+        :param params: a dictionary of parameters.  Some of these can be used
+            to suggest a filename.
```

Comment:
Thanks.  I had thought I was going to add width, height, etc. to the filename, but decided it was clearer not to do so.